ground/uavobjgenerator/main.cpp

```diff
@@ -29,7 +29,7 @@
 #include <QStringList>
 #include <iostream>
 
-#include <unistd.h>	/* For symlink(2) */
+//#include <unistd.h>	/* For symlink(2) */
```

Comment:
It can be removed.  In an earlier version of the fixes I used symlink(2) and I guess this crept into the PR.